public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -43,13 +44,27 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
     <Page navModel={navModel} pageNav={pageNav} layout={PageLayoutType.Custom}>
       {editPanel && <editPanel.Component model={editPanel} />}
       {!editPanel && (
-        <CustomScrollbar autoHeightMin={'100%'}>
-          <div className={styles.canvasContent}>
-            <NavToolbarActions dashboard={model} />
-            {controls && <controls.Component model={controls} />}
-            {isEmpty ? emptyState : withPanels}
-          </div>
-        </CustomScrollbar>
+        <div
+          className={cx(
+            styles.pageContainer,
+            controls && !scopes && styles.pageContainerWithControls,
+            scopes && styles.pageContainerWithScopes,
+            scopes && isScopesExpanded && styles.pageContainerWithScopesExpanded
+          )}
+        >
+          {scopes && <scopes.Component model={scopes} />}
+          <NavToolbarActions dashboard={model} />
+          {controls && (
+            <div
+              className={cx(styles.controlsWrapper, scopes && !isScopesExpanded && styles.controlsWrapperWithScopes)}
+            >
+              <controls.Component model={controls} />
+            </div>
+          )}
+          <CustomScrollbar autoHeightMin={'100%'} className={styles.scrollbarContainer}>
+            <div className={styles.canvasContent}>{isEmpty ? emptyState : withPanels}</div>
+          </CustomScrollbar>
+        </div>
```

Comment:
@bfmatei do you remember why you moved the page scrollbar from page level to just panels? 

he controls are sticky, it's better if the whole page can be scrolled + sticky header (for full page screenshots and and makes it easier to disable sticky controls on mobile with simple media query)

Reply:
There's a whole back story to it but there's also some good news about it.

The rationale was: the scope selector was living on the same level as the filters but it also had the suggested dashboards panel that was acting like a sidebar. When the suggested dashboards panel was expanded, it was fixed on the screen and this was the only way to actually keep it fixed while also keeping the controls sticky.

The good news about it tho', I just merged [this PR](https://github.com/grafana/grafana/pull/90136) that moves the scope selector outside of the dashboard itself. Hence, this workaround is not needed anymore.